security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc

```diff
@@ -27,6 +27,8 @@ The Compliance Operator might report incorrect results on some managed platforms
 
 include::modules/compliance-supported-profiles.adoc[leveloffset=+1]
 
+include::modules/compliance-profile-types.adoc[leveloffset=+2]
+
 [id="additional-resources-compliance-operator-"]
 [role="_additional-resources"]
 == Additional resources
```

Comment:
I think maybe this "Additional resources" section can be removed since it seems to point to the content immediately above it.